qa/suites/upgrade/quincy-x/filestore-remove-check/2 - upgrade.yaml

```diff
@@ -3,14 +3,13 @@ meta:
    install upgrade ceph/-x on cluster
    restart : mons, osd.*
 tasks:
+- print: "**** start install.upgrade of nodes"
 - install.upgrade:
-    mon.a:
-- exec:
-    osd.0:
-      - ceph osd require-osd-release quincy
+    all:
 - print: "**** done install.upgrade of nodes"
+- print: "**** start ceph.restart of all osds"
 - ceph.restart:
-    daemons: [mon.a,mgr.x,osd.0,osd.1,osd.2]
+    daemons: [osd.0,osd.1,osd.2]
     mon-health-to-clog: false
     wait-for-healthy: false
     wait-for-osds-up: false
```

Comment:
I don't get how we're waiting _the osds down_. Would you mind to clarify?

Reply:
@rzarzynski yes, the commit message is not clear enough.. i made the changes.
we didn't wait for the osd to go down before we checked the messages.

Reply:
It's still confusing. Let's schedule a quick call to go over this.